docs/meetings/2023/TechArea20231107.md

```diff
@@ -39,15 +39,31 @@ Remaining OSG 23 items <https://opensciencegrid.atlassian.net/browse/SOFTWARE-55
 -   Figure out if we need VOMS for EL9/an update to EL8
 -   AI (Tim): Update tarball scripts to use `23-main` (SOFTWARE-5713)
 -   AI (BrianL): Finish transition of internal containers to OSG 23
--   AI (Matt): Write signed OSG 23 RPMs to disk and turn strict signature checking on for 23-development
 
 ### Discussion
 
-None this week
+-   How should we manage containers produced by OSG Staff?
+    -   We should separate needs/reqs for containers that are built off of the OSG Software pipeline (dev -> testing ->
+        release Yum repos) vs everything else
+    -   Ideally, for everything else, we'd have a GitHub workflow template or action and some convention for tagging as
+        well as immutability/retention policies
+    -   Repo sprawl is a concern for security and management.
+        Harbor creds are per organization so it may be prudent to separate out mostly-internally facing containers to a
+        separate project, e.g. `osg-internal`
+    -   Consolidation into the `images` repo doesn't work well for many images right now, especially those with "large"
+        code bases as it makes dev (think tracking issues/PRs) more difficult.
+        We should think of the SVN packaging repo as an analog here.
+        We could add some additional flexibility to the `images` repo that would allow per-image references to external,
+        trusted repos and usage of build processes that aren't so tightly tied to the Software release process.
+    -   For the time being for the `xrootd-monitoring-collector`, we should just use a custom GitHub Action
+-   Release of scitokens-cpp is in bodhi and needs karma. Ready for TimT tobuild for Debian
+    -   EPEL7: https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2023-f0f162d810
+    -   EPEL8: https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2023-f9dc18f284
+    -   EPEL9: https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2023-2dfb911a54
 
 ### Support Update
 
-None this week
+-   cvmfs-x509-helper issue with the latest version that causes the client to hang. DaveD is on it.
```

Comment:
I think it's a CVMFS issue, not a cvmfs-x509-helper issue.

Reply:
Fixed, thanks!